oozie/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
```

Comment:
Is this wrapped at this column length just because it comes from a project with 80 cols? It seems like it would fit in a single 120-column line, no?